nsq/reader.py

```diff
@@ -476,6 +499,7 @@ def connect_to_nsqd(self, host, port):
         conn.on('close', self._on_connection_close)
         conn.on('ready', self._on_connection_ready)
         conn.on('message', self._on_message)
+        conn.on('message_timeout', self._on_message_timeout)
```

Comment:
I don't see `AsyncConn` propagating this event, so I don't think this is listening to anything.  Maybe `_on_message_timeout` should be implemented at the `AsyncConn`, not `Reader`, level?